samples/app_event_manager/sample.yaml

```diff
@@ -3,21 +3,21 @@ sample:
   name: Application Event Manager sample
 common:
   harness: console
-  integration_platforms:
-    - qemu_cortex_m3
-    - nrf52dk_nrf52832
-    - nrf52840dk_nrf52840
-    - nrf9160dk_nrf9160_ns
-    - nrf21540dk_nrf52840
   harness_config:
-    type: multi_line
     ordered: false
     regex:
       - "config_event"
       - "measurement_event"
       - "control_event"
       - "ack_event"
       - "Average value3: 45"
+    type: multi_line
+  integration_platforms:
+    - qemu_cortex_m3
+    - nrf52dk_nrf52832
+    - nrf52840dk_nrf52840
+    - nrf9160dk_nrf9160_ns
+    - nrf21540dk_nrf52840
 tests:
-  samples.app_event_manager:
+  sample.event_manager:
```

Comment:
Is name change app_event_manager/event_manager necessary?

Reply:
It was misspelling. Done.